index.html

```diff
@@ -602,6 +602,9 @@ <h2 slot=header>
           <a href=/examples/input-output.html>&lt;input-output&gt;</a>
 
         <li>
+          <a href=/examples/to-do>&lt;to-do&gt;</a>
```

Comment:
@brandondees HOW it does it is the following `DOMParser` logic that runs. It's different per element but this is the spec rules for `<p>` /cc @tmornini 

![capture d ecran 2017-07-07 a 04 52 18](https://user-images.githubusercontent.com/38223/27950653-226700c8-62d0-11e7-9908-9c8a0a8b04cf.png)